pages/_app.tsx

```diff
@@ -54,7 +54,7 @@ const SlopeDemo = ({ Component, pageProps }: AppProps) => {
       <Head>
         <link rel="icon" type="image/x-icon" href="/images/favicon.ico" />
         <title>Slope Checkout Demo</title>
-        <script async src={`${process.env.NEXT_PUBLIC_CHECKOUT_HOST}/slope.min.js`} />
+        <script async src={`${process.env.NEXT_PUBLIC_CHECKOUT_HOST}/slope.min.js?t=${Date.now()}`} />
```

Comment:
Do we ideally fix this on the `checkout/frontend` repo? Wouldn't this mean that any one of our merchant integrations today (eg Plastiq) would throw an error on the checkout page if the customer decided to refresh?

Reply:
It wouldn't break any existing merchants since they are not using any of the new code. Aka they are not calling `initSlopeJs` yet. If they want to, they should explicitly put in a new param like `t=new` so it would clear browser cache.

Reply:
Ideally the Slope.js that we serve at `<script src="https://checkout.slope.so/slope.min.js"></script>` should provide non-caching headers. That way we can update it any time and clients would always fetch newest version.